common/script/content.coffee

```diff
@@ -55,6 +55,24 @@ api.mystery =
   wondercon: {start:'2014-03-24',end:'2014-04-01'} # not really, but the mechanic works
 _.each api.mystery, (v,k)->v.key = k
 
+equipmentList =
```

Comment:
I'm not sure I want it to focus on the gem-purchasable nature of the items because that would appear to exclude future items that might not be purchasable with gems. The primary purpose for this is to allow consistent, complete handling of all equipment types, and while the only location where that's needed now might be in the gem purchasable code, there could be other code in future that's not related to gem purchases.